docs/adapters/sequelize.md

```diff
@@ -0,0 +1,83 @@
+---
+id: sequelize
+title: Sequelize Adapter
```

Comment:
```suggestion
title: Sequelize
```

I would like to remove the `Adapter` wording from the others as well, seems ambiguous